components/org.wso2.carbon.identity.conditional.auth.functions.common/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/common/internal/FunctionsServiceComponent.java

```diff
@@ -56,4 +57,27 @@ protected void unsetIdentityGovernanceService(IdentityGovernanceService identity
         }
         FunctionsDataHolder.getInstance().setIdentityGovernanceService(null);
     }
+
+    @Reference(
+            name = "secret.config.manager",
+            service = SecretResolveManager.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetSecretConfigManager"
+    )
+    protected void setSecretConfigManager(SecretResolveManager secretConfigManager) {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Secret Config Manager is set form functions");
```

Comment:
looks like there's a typo.
Also if the intention is to say that "SecreResolveManager" is set to be referred from the OSGI service class of the conditional auth functions module it should be mentioned so that it's easy to understand the service class just by looking the log.
May be it should say "FunctionsService" or "conditional auth functions service"

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/17a843c6983470ae35ddf181a32231b3390adbf8